### PR TITLE
feat(TPSVC-8903): add action btn to HeaderBar

### PR DESCRIPTION
**Pull request by mkondzera**
**What is the problem this PR is trying to solve?**
We want to have a call to action button on header, according to this item https://jira.talendforge.org/browse/TPSVC-8903
**What is the chosen solution to this problem?**
Add call to action button and callToAction prop to show this button.
**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

### CODE REVIEW

File: packages/components/src/HeaderBar/HeaderBar.scss [hunk 1]
```diff
@@ -78,6 +78,18 @@ $background-color-on-hover: shade($brand-primary, 30);
 			font-size: $font-size-base;
 			text-transform: uppercase;
 		}
+
+		.tc-header-bar-call-to-action {
+			:global(.btn):global(.btn-info):global(.btn-inverse) {
```
Comment (ncomont) [inline packages/components/src/HeaderBar/HeaderBar.scss]: I would prefer something like that, WDYT ?

```css
:global {
    .btn.btn-info.btn-inverse {
        // ...
    }
}
```

File: packages/components/src/HeaderBar/HeaderBar.scss [hunk 2]
```diff
@@ -78,6 +78,18 @@ $background-color-on-hover: shade($brand-primary, 30);
...
+				background-color: $white;
+				border: none;
+				margin-right: 15px;
```
Comment (ncomont) [inline packages/components/src/HeaderBar/HeaderBar.scss]: Please use SASS variable instead

File: packages/components/src/HeaderBar/HeaderBar.scss (hunk 1 above)
Comment (mkondzera) [reply packages/components/src/HeaderBar/HeaderBar.scss]: I think it's less verbal and more readable, without redundant :global

File: packages/components/src/HeaderBar/HeaderBar.component.js [hunk 3]
```diff
@@ -87,6 +87,30 @@ function Environment({ getComponent, ...props }) {
 	);
 }
 
+function CallToAction({ getComponent, t, ...props }) {
+	const actionProps = {
+		bsStyle: 'info',
+		className: 'btn-inverse',
+		label: t('HEADERBAR_ACTION', { defaultValue: 'Subscribe now' }),
```
Comment (lmaillet) [inline packages/components/src/HeaderBar/HeaderBar.component.js]: `label` for the Action comes from the props, so no need for default value.
And looks like `t` is not necessary.

**Pull request merged**